packages/subgraph/test/helpers/helpers.ts

```diff
@@ -136,11 +137,20 @@ export const subgraphRequest = async <T>(
     variables?: { [key: string]: any }
 ): Promise<T> => {
     try {
-        const response = await request<T>(
+        const response = await request<T & ISubgraphErrors>(
             "http://localhost:8000/subgraphs/name/superfluid-test",
             query,
             variables
         );
+
+        if (response.errors) {
+            throw new Error(
+                `Failed call to subgraph with query ${query}, error thrown: ${JSON.stringify(
```

Comment:
The `query` is not always a string, it's often a GraphQL AST object which is JSON basically. Changing this to "Failed call to subgraph with query ${JSON.stringify(query)}..." is a simple fix.